docs/build.sbt

```diff
@@ -111,7 +111,7 @@ lazy val immutables = ProjectRef(parentDir, "immutables")
 def forkedTests: Seq[Setting[_]] = Seq(
   fork in Test := true,
   concurrentRestrictions in Global += Tags.limit(Tags.Test, 1),
-  javaOptions in Test ++= Seq("-Xms256M", "-Xmx512M", "-Dakka.test.single-expect-default=15s"),
+  javaOptions in Test ++= Seq("-Xms256M", "-Xmx512M"),
```

Comment:
This was not kicking in. I've tried `300ms` instead of `15s` and tests still passed.

I even tried few combinations of prefixed settings like:

`  javaOptions in Test ++= Seq("-Xms256M", "-Xmx512M", "-Dplay.akka.test.single-expect-default=450ms"),`

I suspect the right preffix would make this work but I find the final solution of on-location setting overwrite a lot clearer (see below).